addons/characters/CfgVehicles.hpp

```diff
@@ -0,0 +1,69 @@
+#define UNLOCK_MALARIA_INFECTED_CIVILIAN  \
+    editorSubcategory = "EdSubcat_Personnel_MalariaInfected"; \
+    scope = 2; \
+    scopeCurator = 2; \
+    class EventHandlers { \
+        init = "(_this select 0) setIdentity selectRandom [ \
+            'BIS_Ambient01_sick', \
+            'BIS_Ambient02_sick', \
+            'BIS_Ambient03_sick', \
+            'BIS_Arthur_sick', \
+            'BIS_Howard_sick', \
+            'BIS_John_sick', \
+            'BIS_Lucas_sick', \
+            'BIS_Renly_sick' \
+        ]; \
+        (_this select 0) setDamage 0.45"; \
+    }
+
+class CfgVehicles {
+    class C_Man_casual_1_F_afro;
+    class C_Man_casual_1_F_afro_sick: C_Man_casual_1_F_afro {
+       UNLOCK_MALARIA_INFECTED_CIVILIAN;
+    };
+
+    class C_Man_casual_3_F_afro;
+    class C_Man_casual_3_F_afro_sick: C_Man_casual_3_F_afro {
+       UNLOCK_MALARIA_INFECTED_CIVILIAN;
```

Comment:
The class is missing an `author` entry. Keep in mind that `author` can not be inherited (is ignored when inherited), so these classes will currently display as "unknown community author" (or something similar), which is funny, because it's super generic, yet still completely wrong.

I'd add to the UNLOCK_xxx macro:
```cpp
        author = "$STR_A3_Bohemia_Interactive";
```